resources/services/backup/global_settings.go

```diff
@@ -52,7 +52,7 @@ func fetchBackupGlobalSettings(ctx context.Context, meta schema.ClientMeta, pare
 		o.Region = c.Region
 	})
 	if err != nil {
-		if client.IgnoreAccessDeniedServiceDisabled(err) {
+		if client.IgnoreAccessDeniedServiceDisabled(err) || client.IsAWSError(err, "ERROR_9601") /* "Your account is not a member of an organization" */ {
```

Comment:
could this happen for other services? should we make a generic access denied?

Reply:
we have a generic access denied but this serms to be special